torch/csrc/utils/tensor_new.cpp

```diff
@@ -330,6 +330,9 @@ Tensor internal_new_from_data(
     tensor = tensor.to(device, inferred_scalar_type, /*non_blocking=*/false, /*copy=*/false);
   }
 
+  // torch.jit.trace will continue to trace out `.to()` instead of `.lift()`, since
+  // changing it is BC-breaking.
```

Comment:
so we should actually change this message to be FC breaking, because we can change in 2 weeks if we wanted to right?

Reply:
Yep that's right